generators/app/templates/src/StarterKit/StarterKit.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <VersionPrefix>1.0.0-BUILDNUMBER</VersionPrefix>
+    <VersionPrefix>2.0.0-BUILDNUMBER</VersionPrefix>
```

Comment:
Deze versienr mag op 1 blijven staan (is de versienr van de gegenereerde applicatie)